### DISCUSSION TITLE
怎么样让 x6 中的 scale 属性放大节点 ,而不是放大节点的包围框呢?

### DISCUSSION THREAD

**Discussion by huaguanjun** [antvis/X6#3741] [Q&A] [answered]
怎么样让 x6 中的 scale 属性放大节点 ,而不是放大节点的包围框呢?

**Comment by NewByVector** [comment]
你是想节点所有内容同步放大是吧，这个需要节点内元素时按照相对位置和相对尺寸布局。

**Comment by huaguanjun** [comment, marked as answer]
有没有具体操作的事例啊



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: vector ***@***.***&gt;
发送时间: 2023年7月10日 11:30
收件人: antvis/X6 ***@***.***&gt;
抄送: huaguanjun ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [antvis/X6] 怎么样让 x6 中的 scale 属性放大节点 ,而不是放大节点的包围框呢? (Discussion #3741)





 
你是想节点所有内容同步放大是吧，这个需要节点内元素时按照相对位置和相对尺寸布局。
